doc/main/reference/try_put_and_wait.rst

```diff
@@ -0,0 +1,324 @@
+.. _try_put_and_wait:
+
+Waiting for single messages in Flow Graph
+=========================================
+
+.. contents::
+    :local:
+    :depth: 1
+
+Description
+***********
+
+This feature adds a new ``try_put_and_wait`` to the receiving nodes in the Flow Graph.
+This function allows to wait until completion of single message, submitted to ``try_put_and_wait`` as an input by the Flow Graph in
+opposite to ``graph.wait_for_all()`` function that waits for completion of all tasks in the graph.
+
+The ``node.try_put_and_wait(msg)`` is performing ``node.try_put(msg)`` on the node and blocks until the work on ``msg`` is completed.
```

Comment:
```suggestion
``node.try_put_and_wait(msg)`` performs ``node.try_put(msg)`` on the node and blocks it until the work on ``msg`` is completed.
```

Reply:
Partially fixed, I would prefer to use `waiting until` instead of `blocks` since it is not clear what "blocking the function" means and I don't want to use terminology of blocking the thread here.